wrappers/python/openmm/app/forcefield.py

```diff
@@ -436,7 +436,16 @@ def registerAtomType(self, parameters):
         """Register a new atom type."""
         name = parameters['name']
         if name in self._atomTypes:
-            raise ValueError('Found multiple definitions for atom type: '+name)
+            #  allow multiple registrations of the same atom type provided the definitions are identical
+            existing = self._atomTypes[name]
+            if (
+                existing.atomClass != parameters['class']
+            ) or (
+                existing.mass != float(parameters['mass'])
+            ) or (
+                existing.element.symbol != parameters['element']
+            ):
```

Comment:
I tried to keep the change minimal, but there isn't a really pretty way to string together three expressions like is needed here. Python should short-circuit on the first one that evaluates falsy, so maybe there's a better order than what I happened to think of.

Reply:
It might be safer to write it like this:

```python
if existing.atomClass == parameters['class'] and existing.mass == float(parameters['mass']) and existing.element.symbol == parameters['element']:
    return
```

That way it will exit leaving all existing definitions intact, instead of overwriting them with new definitions.  That seems less likely to cause problems, especially if code related to atom types changes in the future.